include/El/blas_like/level1/Copy/util.hpp

```diff
@@ -250,19 +250,17 @@ void InterleaveMatrix(
 {
     if (colStrideA == 1 && colStrideB == 1)
     {
-        H_CHECK_CUDA(
-            cudaMemcpy2DAsync(B, rowStrideB*sizeof(T),
-                              A, rowStrideA*sizeof(T),
-                              height*sizeof(T), width,
-                              cudaMemcpyDeviceToDevice,
-                              syncInfo.stream_));
+        gpu::Copy2DIntraDevice(A, rowStrideA,
+                               B, rowStrideB,
+                               height, width,
+                               syncInfo);
     }
     else
     {
         hydrogen::Copy_GPU_impl(height, width,
                                 A, colStrideA, rowStrideA,
                                 B, colStrideB, rowStrideB,
-                                syncInfo.stream_);
+                                syncInfo.Stream());
```

Comment:
It's a bit odd that this still takes a stream, but other calls are taking the full `syncInfo`.

Reply:
Will fix.

Reply:
Fixed. Thanks.